### ISSUE TITLE
[BUG] CV HUD Element not functional

### ISSUE THREAD

**Issue by Brimcon** [p4535992/conditional-visibility#40]
**Module Version:** v0.6.4 (First seen in 0.6.0

**Describe the bug**
Clicking on the eyeball on the Token Hud does nothing.

**To Reproduce**
1. Enable CV HUD element
2. Click on the Eyeball

**Expected behavior**
Expected a window to open up

**Screenshots**
![image](https://user-images.githubusercontent.com/7198283/171047998-60890fbc-8ef3-4910-ba1b-68cec557a905.png)

**Browser:**
Electrum Local and Chrome Remote

**Foundry Version:** 9.269

**Game System:** DnD5e 1.6.1

**Additional context**
```js
Roll#evaluate is becoming asynchronous. In the short term you may pass async=true or async=false to evaluation options to nominate your preferred behavior.
```
Is printed twice in the console upon clicking the element.

**Comment by p4535992** [comment]
Should be fixed on 0.6.5 , ty i know about that warning for now i'm focused on fixing the bugs.

**Comment by Brimcon** [comment]
Fixed! Thank you.
